src/components/table/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC } from 'react';
+import {
+  Table as ChakraTable,
+  TableCellProps as ChakraTableCellProps,
+  TableColumnHeaderProps as ChakraTableColumnHeaderProps,
+} from '@chakra-ui/react';
+export type {
+  TableBodyProps,
+  TableCaptionProps,
+  TableColumnGroupProps,
+  TableColumnProps,
+  TableFooterProps,
+  TableHeaderProps,
+  TableRootProps,
+  TableRowProps,
+  TableScrollAreaProps,
+} from '@chakra-ui/react';
+
+const {
+  Body,
+  Caption,
+  Column,
+  ColumnGroup,
+  Footer,
+  Header,
+  Root,
+  Row,
+  ScrollArea,
+} = ChakraTable;
+
+Body.displayName = 'Table.Body';
+Caption.displayName = 'Table.Caption';
+Column.displayName = 'Table.Column';
+ColumnGroup.displayName = 'Table.ColumnGroup';
+Footer.displayName = 'Table.Footer';
+Header.displayName = 'Table.Header';
+Root.displayName = 'Table.Root';
+Row.displayName = 'Table.Row';
+ScrollArea.displayName = 'Table.ScrollArea';
```

Comment:
This mitigates the issue with how components are visible in react component tree inspector and storybook code view